dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe

```diff
@@ -46,6 +45,14 @@ INSTALL()
 {
 	LUAVERSION=$(lua -v | sed 's/Lua //' | head -c3)
 	make LUA_LIBDIR="$libDir/lua/$LUAVERSION" install
+
+#	copy the doc files
+#	mkdir -p $docDir
+#	cp -r doc/* $docDir
+
+#	copy the header file
+#	mkdir -p mkdir -p $includeDir
+#	cp src/lfs.h $includeDir
```

Comment:
Not sure if these are needed, no package has been depending on them ...

Reply:
If "make install" does not install them, it means upstream does not think they are needed. So, don't copy them.